File: frontEnd/src/types/dto/stepDTO/OpenPageStepDTO.ts
// 打开页面操作
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";

export class OpenPageStepDTO extends StepBaseDTO {
    stepType: string = 'OPEN_PAGE'; // 步骤类型
    url: string = ''; // 页面地址
    recover: number = 0;// 是否覆盖输入

}

File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/OpenPageStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// 打开页面操作

import lombok.Data;

@Data
public class OpenPageStepDTO extends StepBaseDTO {
    private String stepType; // 步骤类型
    private String url; // 页面地址
    private Integer recover;// 是否覆盖输入

}
